clang/test/Driver/windows-compat-sycl-args.cpp

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang -### --sycl -c %s 2>&1 | FileCheck %s
+// RUN: %clang -### -fsycl-device-only -fsycl %s 2>&1 | FileCheck %s
+// CHECK: -fdelayed-template-parsing
+// CHECK: -fms-compatibility
+// REQUIRES: system-windows
```

Comment:
Please, set the triple with Windows OS component to make test OS agnostic.

Reply:
@mdtoguchi , how do I set the triple with the driver so it is OS agnostic?

Reply:
You can add -target x86_64-unknown-windows-msvc (or something like that) to the command line.  The driver will behave according to the target specified.